docs/usage/polyfill.md

```diff
@@ -42,14 +42,32 @@ polyfills are loaded first:
 import "babel-polyfill";
 ```
 
-With `webpack.config.js`, add `babel-polyfill` to your entry array:
+When using Webpack there are multiple ways to include Babel Polyfill. 
```

Comment:
Let's convert "Babel Polyfill" to `babel-polyfill` here and everywhere else.

Reply:
Disagree.  

_Babel Polyfill_ is the name of the game.  
Once you change it to `babel-polifill` it looks like name of the package to install.  
Reader comes to the docs, sees `babel-polifill` and just hits `npm i babel-polifill` which in near future will be wrong thing to do.  Thus it would need yet another change again to `@babel/polyfill`.

If the name *Babel Polyfill* was left then we could avoid all that. The reader would just refer to https://babeljs.io/docs/usage/polyfill/#installation where the correct package name is noted. I suggest not mixing "name" with "current package name".